### DISCUSSION TITLE
After the app has been running for a period of time, some routes will lose response and will return 304 when accessed

### DISCUSSION THREAD

**Discussion by cfftony** [vercel/next.js#55893] [Help] [answered]
### Summary

After the app has been running for a period of time, some routes will lose response and will return 304 when accessed (normally it should be 200). 
Killing pid and restarting node has no effect.
We must recompile and then start the service to restore the service.
Next.js 13.5
Nodejs 18.18.0

### Additional information

```js
curl -i 'http://127.0.0.1:3100/news'
HTTP/1.1 304 Not Modified
set-cookie: User-UUID=34d041f3-dd64-4855-b00e-c93978bbc893; Path=/; Expires=Tue, 24 Sep 2024 04:31:55 GMT
Vary: RSC, Next-Router-State-Tree, Next-Router-Prefetch, Next-Url, Accept-Encoding
x-nextjs-cache: STALE
Cache-Control: s-maxage=600, stale-while-revalidate
ETag: "vz3aqh8r2g8bf0"
Content-Type: text/html; charset=utf-8
Content-Length: 388737
Date: Sun, 24 Sep 2023 03:31:55 GMT
Connection: keep-alive
Keep-Alive: timeout=5
```

**Comment by mih0vil** [comment]
I am experiencing the same problem with NextJS 13.5 (same version). This is production version started with `next start`

**Comment by stcore** [comment]
Please help, the routing loses response in production after a moment of execution despite giving a status of 200. You have to reload the page to continue browsing!

**Comment by mih0vil** [comment, marked as answer]
it seems to me that disabling etag generation helps:
https://nextjs.org/docs/app/api-reference/next-config-js/generateEtags
Does it work for you too?

**Reply by stcore** [reply]
It seems to work, so far it's going well, I'm still doing tests to detect if the navigation is still interrupted

**Reply by cfftony** [reply]
It seems to work！Thanks
